react/tic-tac-toe/src/app/components/AuthRoute/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { push } from 'connected-react-router';
+import { Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { TOKEN } from '../../../constants/autentications';
+import { LOGIN } from '../../../constants/routes';
+import actionsCreators from '../../../redux/login/actions';
```

Comment:
I'd rename `actionsCreators` to `actionsAuth`. Also into `'../../../redux/login/actions'`